app/views/shared/_records_table.html.erb

```diff
@@ -41,7 +41,7 @@
               <td class="center_text"><%= record.revision_id %></td>
               <td><%= record.recordable.short_name %></td>
               <!-- Only Version ID should be loaded -->
-              <td class="center_text"><%= record.record_datas.first.value%></td>
+              <td class="center_text"><%= record.version%></td>
```

Comment:
Here I'm using the new version property, rather than assuming the first record data MUST be a version and ONLY a version.